pkg/scheduler/plugins/reservation/hook.go

```diff
@@ -232,6 +234,27 @@ func preparePreFilterNodeInfo(nodeInfo *framework.NodeInfo, pod *corev1.Pod, mat
 			*podInfo = *newPodInfo
 		}
 	}
+
```

Comment:
Add some comments here to describe why we need to do this

Reply:
+1